oio/directory/meta1.py

```diff
@@ -53,6 +53,11 @@ def _apply_link_services(self, moved_ok, **kwargs):
                     service_type=service_type, cid=cid,  replace=True,
                     services=dict(host=','.join(peers), type=service_type,
                                   args=args, seq=seq))
+                if service_type == 'meta2' and kwargs.get('src_service'):
+                    self.rdir.meta2_index_delete(
+                        volume_id=kwargs.get('src_service'),
+                        container_id=cid
+                    )
```

Comment:
I suggest adding a note telling that we will be able to remove this part when either:
- meta1 `account.services` events contain the list of services that have been removed;
- or meta2 sends `storage.container.deleted` events even when called at the sqliterepo level.

Reply:
Added in  f5c557b